### ISSUE TITLE
Welcome panel remains visible to screen readers

### ISSUE THREAD

**Issue by LJWatson** [jsbin/jsbin#2841]
When the welcome panel is dismissed, it disappears visually but remains present for screen reader users (even when the page is reloaded). Found in Windows 10 + Jaws/FF and Narrator/Edge.

Once dismissed the containing `<div>` seems to have display:none; set. This should be sufficient to hide the panel both visually and programmatically, so something else must be getting in the way.

A quick/dirty workaround would be to put aria-hidden="true" on the container once the panel has been dismissed.

**Comment by remy** [comment]
👍 If I recall correctly, the panel is actually just shifted up out of view to the user, so putting the aria hidden attribute sounds like the right approach.

You mention that this is the "dirty workaround", is it technically wrong to use this attribute (I'm only ask so I can learn more!).

Thank you again.

**Comment by LJWatson** [comment]
It would be better to use HTML hidden I think. This encompasses the benefits of both display:none; and aria-hidden, each of which has <a href="https://www.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/">drawbacks</a> based on different browsers and/or situations.

The hidden attribute effectively applies display:none; to the thing, removes it from the accessibility tree and from the keyboard tab order. It's also good cross-browser and AT.

**Comment by remy** [comment]
Thanks understood. To start with, I'll likely drop in the aria hidden attribute and then look at adding the full display:none (the reason behind this is that when the welcome panel is fully hidden, it'll shift up the entire layout, which requires a bunch more work to get it right).
